src/Symfony/Component/Routing/Tests/Fixtures/controller_aware.yml

```diff
@@ -0,0 +1,9 @@
+app_homepage:
+    path: /
+    controller: AppBundle:Homepage:show
+
+app_blog:
+    path: /blog
+    controller: AppBundle:Homepage:show
```

Comment:
@ogizanagi, I think you're right. I've changed the code to throw the exception when both controller attribute and _controller default are set.